test.cljs.edn

```diff
@@ -5,6 +5,13 @@
 
   ;; uncomment to launch tests in a headless environment
   ;; you will have to figure out the path to chrome on your system
-  :launch-js ["chromium" "--headless=new" "--disable-gpu" "--repl" :open-url]
+  :launch-js ["chromium"
+              "--headless=new"
+              "--disable-gpu"
+              "--no-sandbox"
+              "--disable-setuid-sandbox"
+              "--password-store=basic"
```

Comment:
Make sure to use the old basic password store instead of any poxy OS keychain (this will stop you from getting little auth boxes when you run the tests on your mac)